ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Decision/Add.cshtml.cs

```diff
@@ -127,8 +132,13 @@ private void PopulateCreateDecisionDtoFromRequest()
 			var dtr_month = Request.Form["dtr-month-request-received"];
 			var dtr_year = Request.Form["dtr-year-request-received"];
 			var dtString = $"{dtr_day}-{dtr_month}-{dtr_year}";
-			var dateStarted = DateTimeHelper.TryParseExact(dtString, out DateTime parsedDate) ? parsedDate : (DateTime?)null;
-						
+			var isValidDate = DateTimeHelper.TryParseExact(dtString, out DateTime parsedDate);

```

Comment:
Should we be using a culture here - does this work regardless of the local machine settings (US date format / UK date format)?